sylvia-derive/src/parser.rs

```diff
@@ -342,3 +344,89 @@ pub fn parse_struct_message(source: &ItemImpl, ty: MsgType) -> Option<(&ImplItem
     }
     Some((method, msg_attr))
 }
+
+#[derive(Debug)]
```

Comment:
This is more about defining custom attr, it's form and providing a way to load it. Yes it's an intermediate step which I felt had to be extracted as it represents some functionality. Otherwise next PR would be too big